scripts/build.sh

```diff
@@ -1,5 +1,9 @@
 #!/usr/bin/env bash
 
+if [[ "$(git tag --points-at HEAD 2>/dev/null)"=="v*" ]]; then
+	touch prerelease.txt
```

Comment:
Please make this work even if the script is run from the wrong directory. You ran for example just move the line containing `cd $(dirname "$0")/..` upwards.

Reply:
Fixed.